packages/wonder-blocks-button/src/components/button-core.js

```diff
@@ -381,14 +380,26 @@ const _generateStyles = (color, kind, light, iconWidth, size) => {
                     height: 2,
                     width: `calc(100% - ${iconWidth}px)`,
                     right: 0,
-                    bottom: "calc(50% - 11px)",
+                    bottom: -1,
                     background: light ? fadedColor : activeColor,
                     borderRadius: 2,
                 },
             },
             disabled: {
                 color: light ? fadedColor : offBlack32,
                 cursor: "default",
+                ":focus": {
+                    ":after": {
+                        content: "''",
+                        position: "absolute",
+                        height: 2,
+                        width: `calc(100% - ${iconWidth}px)`,
+                        right: 0,
+                        bottom: `calc(50% - 11px)`,
```

Comment:
**question:** could we do `bottom: -1` here as well so that it's consistent with the `active` style?

Reply:
ahh good question. I tried to do that originally, but the issue is that in that case the underline is applied to the `button` instead of the internal `span`, so the line is shown too far from the `span` element. Note that this is a specific case for `tertiary` buttons. I'll add a comment there to explain all of this.